mods/ra/maps/mousetrap/rules.yaml

```
@@ -0,0 +1,142 @@
+World:
+	LuaScript:
+		Scripts: campaign.lua, utils.lua, Mousetrap.lua
```

```suggestion
		Scripts: campaign.lua, utils.lua, mousetrap.lua
```

Linux is case sensitive and will crash.

---

Thank you for the correction. Fixed.